apps/shared/zoom_parameter_controller.cpp

```diff
@@ -59,9 +59,9 @@ int ZoomParameterController::ContentView::numberOfSubviews() const {
 View * ZoomParameterController::ContentView::subviewAtIndex(int index) {
   assert(index >= 0 && index < 2);
   if (index == 0) {
-    return m_curveView;
+    return &m_legendView;
```

Comment:
This is indeed a good trick to fix the bug. A comment explaining why this order (because, now, it matters) would be nice!

Reply:
The subview are redrawn one after the other, so by changing the subview order, the legendview will be redraw before the curveview, and the legendview, which wasn't redrawn, so shown like a sort of glitch isn't anymore.

Reply:
I meant in the code :)
In the same way it is done in apps/calculation/history_view_controller.cpp (only the first part of the comment):
```
View * HistoryViewCell::subviewAtIndex(int index) {
  /* The order of the subviews should not matter here as they don't overlap.
   * However, the order determines the order of redrawing as well. For several
   * reasons listed after, changing subview selection often redraws the entire
   * m_scrollableOutputView even if it seems unecessary:
   * - Before feeding new Layouts to ExpressionViews, we reset the hold layouts
   *   in order to empty the Poincare pool and have more space to compute new
   *   layouts.
   * - Even if we did not do that, ExpressionView::setLayout doesn't avoid
   *   redrawing when the previous expression is identical (for reasons
   *   explained in expression_view.cpp)
   * - Because of the toggling burger view, ExpressionViews often have the same
   *   absolute frame but a different relative frame which leads to redrawing
   *   them anyway.
   * All these reasons cause a blinking which can be avoided if we redraw the
   * output view before the input view (starting with redrawing the more
   * complex view enables to redraw it before the vblank thereby preventing
   * blinking).
   * TODO: this is a dirty hack which should be fixed! */
  View * views[3] = {&m_scrollableOutputView, &m_inputView, &m_ellipsis};
  return views[index];
}
```

Reply:
done :+1: